packages/documentation-site/patternfly-docs/content/design-guidelines/styles/typography/typography.js

```diff
@@ -23,172 +22,138 @@ export const TitleLevel = ({ asGrid, styleProps, children, title = '', note = ''
 
 export const styleProps = {
   superHero: {
-    fontWeight: "400",
+    fontWeight: "--pf-t--global--font--weight--body",
     fontWeightText: "400 (medium)",
-    // variableName: "--pf-v6-global--FontSize--4xl",
+    variableName: "--pf-t--global--font--size--4xl",
     fontSize: "36px",
     lineHeight: "1.3",
-    fontFamily: "RedHatDisplay",
+    fontFamily: "--pf-t--global--font--family--200",
```

Comment:
Should this all the `--pf-t--global--font--family--200` be `--pf-t--global--font--family--heading`? (semantic token instead of base token)